website/docs/d/schema_site_vrf_region_cidr_subnet.html.markdown

```diff
@@ -37,9 +38,10 @@ description: |-
 
 ## Attribute Reference ##
 
-* `template_name` - (Optional) Template where Vrf Region Cidr Subnet to be created.
-* `zone` - (Optional) The name of the zone for the region CIDR subnet.
-* `usage` - (Optional) The usage for the region CIDR Subnet.
+* `zone` - (Read-Only) The name of the zone for the region CIDR subnet.
+* `name` - (Read-Only) The name for the region CIDR Subnet.
+* `usage` - (Read-Only) The usage for the region CIDR Subnet.
+* `subnet_group` - (Read-Only) The subnet group for the region CIDR Subnet.
```

Comment:
same descriptions for both resource and datasource attributes might be better.

Reply:
I do not agree with this, since the data source has read-only attributes. In the resource there are restrictions to input defined which do not count for read.

Reply:
ok